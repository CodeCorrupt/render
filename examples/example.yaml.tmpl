{{ .AppName }} - simple go-template files render