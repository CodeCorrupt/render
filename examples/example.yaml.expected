name: render

nested:
  things:
  - one: 1
  - two: 2
  - three: 3

from_file:
  inner:
  - value: 2

from_gzip: "compressed content"

to_gzip: "H4sIAAAAAAAA/w=="